cognite/client/data_classes/shared.py

```diff
@@ -47,3 +47,94 @@ def __init__(self, count: int = None, value: Union[int, str] = None, **kwargs):
         self.value = value
 
     value = CognitePropertyClassUtil.declare_property("value")
+
+
+class Geometry(dict):
+    """Represents the points, curves and surfaces in the coordinate space.
+
+    Args: type (str): The geometry type. One of 'Point', 'MultiPoint, 'LineString', 'MultiLineString', 'Polygon', or 'MultiPolygon'.
```

Comment:
I think it's better to put a sanity check on the `type` arg supplied in this constructor? ValueError if there's some input that you don't want.

Reply:
Good point.

Reply:
@maaland please look into this.

Reply:
Ah I implemented it on Friday but forgot to `git push` 😆